components/Portrait.js

```diff
@@ -35,7 +45,7 @@ const portraitImageRightStyle = css({
 })
 
 const Portrait = ({odd, image, description, name, age, title, email}) => (
-  <div {...portraitStyle}>
+  <div {...portraitStyle} id={slugify(name)}>
```

Comment:
normally I'd use a relative positioned anchor tag (`top: -headerHeight`):
https://github.com/orbiting/republik-frontend/blob/f190f5208d2fc28d6f07e927c85b9c4d854cb36c/components/Faq/List.js#L127
but if a `paddingTop` value of exactly the header height is what you want I'm also fine with the current solution